install_config/configuring_pvc_protection.adoc

```diff
@@ -0,0 +1,59 @@
+[[install-config-configuring-pvc-protection]]
+= Configuring Persistent Volume Claim Protection
```

Comment:
@tmorriso-rh Is this configuration specific to installation time? I'm wondering if this file would be better at home in the Admin Guide. Perhaps even in the Managing Pods section:

https://docs.openshift.com/container-platform/3.7/admin_guide/managing_pods.html

Reply:
Yes, this is specific to installation.